packages/matrix-client-server/src/index.ts

```diff
@@ -85,15 +86,20 @@ export default class MatrixClientServer extends MatrixIdentityServer<clientDbCol
           }
           this.api.get = {
             '/_matrix/client/v3/account/whoami': whoami(this),
-            '/_matrix/client/v3/admin/whois': whois(this)
+            '/_matrix/client/v3/admin/whois': whois(this),
+            '/_matrix/client/v3/user/:userId/account_data/:type':
+              accountDataType(this)
           }
           this.api.post = {
             '/_matrix/client/v3/account/whoami': badMethod,
-            '/_matrix/client/v3/admin/whois': badMethod
+            '/_matrix/client/v3/admin/whois': badMethod,
+            '/_matrix/client/v3/user/:userId/account_data/:type': badMethod
           }
           this.api.put = {
             '/_matrix/client/v3/account/whoami': badMethod,
-            '/_matrix/client/v3/admin/whois': badMethod
+            '/_matrix/client/v3/admin/whois': badMethod,
+            '/_matrix/client/v3/user/:userId/account_data/:type':
+              accountDataType(this)
```

Comment:
I do not understand why we register 2 time the same method that then introduces 2 diffrent behaviours later on.

Suggestion:

 this.api.get = {
 ...
             '/_matrix/client/v3/user/:userId/account_data/:type':
              getAccountDataType(this)
  }
this.api.put = {
...
'/_matrix/client/v3/user/:userId/account_data/:type':
              putAccountDataType(this)
}

Reply:
You're right, I'll separate the two behaviours to make things clearer, thank you !